mne/viz/backends/_pyvista.py

```diff
@@ -773,6 +773,18 @@ def _glyph(dataset, scale_mode='scalar', orient=True, scalars=True, factor=1.0,
     return alg.GetOutput()
 
 
+def _sphere(plotter, center, color, radius):
```

Comment:
Can you add some comment about why this is necessary / `renderer.sphere` did not work?

Reply:
I was able to reproduce a segfault when I was clicking/picking a lot in a very short period of time. I assume here that this is what happens during testing. I compared the 2 functions and the only noticeable difference is the use of a glyph filter before adding a sphere to the scene. So I guess that the pipeline is heavier with glyph and maybe some form of synchronisation could mitigate the issue with `renderer.sphere()`. On the other side, using glyph to only add one sphere looked a bit overkill to me.

Reply:
Okay can you put this in the code comments? At some point we might consider just adding some delay or a `processEvents` after the clicking.

Also I would just set `rtol=1e-6` in to fix https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=7774&view=logs&j=a846d25a-e32c-5640-1b53-e815fab94407&t=f8799db5-b972-5f58-516d-bc88b8357f99&l=50